masu/config.py

```diff
@@ -46,8 +46,11 @@ class Config(object):
     SQLALCHEMY_TRACK_MODIFICATIONS = False
 
     SECRET_KEY = os.getenv('MASU_SECRET_KEY')
-    if not SECRET_KEY:
+    if SECRET_KEY is None:
         raise ValueError('No secret key set for Masu application')
 
     # Logging
     LOG_LEVEL = os.getenv('LOG_LEVEL', 'INFO')
+
+    # Data directory for processing incoming data
+    TMP_DIR = '/var/tmp/masu'
```

Comment:
Based on the FHS, stuff in /var/tmp is temporary, but should persist across reboots where /tmp cannot be expected to persist across reboots.

IMO, using /tmp is fine, but I don't have a strong opinion. So, something to consider, but not a blocker for the PR.

Reply:
I think file download is a stand-in right now for dealing with a stream from S3. My reasoning behind using /var/tmp is that it *would* persist across reboots. This might not work exactly the same in container world (and there isn't currently a persistent volume set here, so I'm not sure we get anything out of this at the moment), but my thinking was we might crash in the middle of processing a file. When we come back to, the first check could be for a file in this directory. If one is there, process it. When we are done processing a downloaded file, we should delete it. It's bare bones disaster recovery.